1-WebApp-OIDC/1-5-B2C/README.md

```diff
@@ -0,0 +1,172 @@
+
+## Scenario
+
+This sample shows how to build a .NET Core 2.2 MVC Web app that uses OpenID Connect to sign in users in **Azure AD B2C**. It assumes you have some familiarity with **Azure AD B2C**. If you'd like to learn all that B2C has to offer, start with our documentation at aka.ms/aadb2c.
+
+![Sign in with Azure AD](ReadmeFiles/sign-in.png)
+
+## How to run this sample
+
+To run this sample:
+
+> Pre-requisites: Install .NET Core 2.2 or later (for example for Windows) by following the instructions at [.NET and C# - Get Started in 10 Minutes](https://www.microsoft.com/net/core). In addition to developing on Windows, you can develop on [Linux](https://www.microsoft.com/net/core#linuxredhat), [Mac](https://www.microsoft.com/net/core#macos), or [Docker](https://www.microsoft.com/net/core#dockercmd).
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```powershell
+git clone https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2.git
+```
+
+> Given that the name of the sample is very long, and so are the name of the referenced NuGet packages, you might want to clone it in a folder close to the root of your hard drive, to avoid file size limitations on Windows.
+
+### Step 2: Get your own Azure AD B2C tenant
+
+If you don't an Azure AD B2C tenant yet, you'll need to create an Azure AD B2C tenant by following [these instructions](https://azure.microsoft.com/documentation/articles/active-directory-b2c-get-started).
+
+### Step 3: Create your own user flow(policy)
+
+This sample uses a unified sign-up/sign-in user flow(policy). Create this policy by following [the instructions here](https://azure.microsoft.com/documentation/articles/active-directory-b2c-reference-policies). You may choose to include as many or as few identity providers as you wish, but make sure **DisplayName** is checked.
+
+If you already have an existing unified sign-up/sign-in user flow(policy) in your Azure AD B2C tenant, feel free to re-use it. No need to create a new one just for this sample.
+
+Copy this policy name, so you can use it in step 5.
+
+### Step 4: Create your own Web app
+
+Now you need to [register your web app in your B2C tenant](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-app-registration#register-a-web-application), so that it has its own Application ID.
+
+Your web application registration should include the following information:
+
+- Enable the **Web App/Web API** setting for your application.
+- Set the **Reply URL** to `https://localhost:44321//signin/B2C_1_sign_up_in` and `https://localhost:44321//signout/B2C_1_sign_up_in`.
+- Copy the Application ID generated for your application, so you can use it in the next step.
+
+### Step 5: Configure the sample with your app coordinates
+
+1. Open the solution in Visual Studio.
+1. Open the `appsettings.json` file.
+1. Find the assignment for `Instance` and replace the value with `<your-tenant-name>.b2clogin.com`
```

Comment:
><your-tenant-name>.b2clogin.com [](start = 66, length = 31)

this isn't very clear...maybe something like "replace the value of <your-tenant-name> with your tenant name" and give an example?